components/brave_wallet_ui/components/extension/transaction-list-item/index.tsx

```diff
@@ -141,26 +140,29 @@ export const TransactionsListItem = ({
   const gasFee = isSolTx ? solEstimatedFee : getTransactionGasFee(transaction)
 
   const transactionDetails = React.useMemo(() => {
-    return spotPriceRegistry
-      ? parseTransactionWithPrices({
+    if (!transactionsNetwork || !transactionAccount || !accounts)
```

Comment:
```suggestion
    if (!spotPriceRegistry || !transactionsNetwork || !transactionAccount || !accounts)
```

Reply:
Added https://github.com/brave/brave-core/pull/19586/commits/de8a47305ae846acfbfed7247e3d7b5a88096ad7
Actually not sure why it is needed. `spotPriceRegistry` is never falsy.